src/assets/Ripple.svg

```diff
@@ -0,0 +1,10 @@
+<svg width="185px"  height="185px"  xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100" preserveAspectRatio="xMidYMid" class="lds-ripple" style="background: none;">
```

Comment:
I think it'd be nice to keep consistent with current naming convention and just use lowercased name. I.e. `ripple.svg`.

Reply:
sure. will change